Fix Windows style path

hiroakis opened the pull request:
This patch will resolve an issue of https://github.com/Songmu/horenso/issues/31. The PR has 3 commits.

- https://github.com/hiroakis/horenso/pull/2/commits/756a56b27beb32e195e74c31c3935ff14210bde0: Make testable for that issue.
- https://github.com/hiroakis/horenso/pull/2/commits/c4777bd08ab2c71e8121aae6a240dfaf62abce83: Testing (has a failure case)
```
horenso = go test -run TestSplitHandlerCmdStr
--- FAIL: TestSplitHandlerCmdStr (0.00s)
    --- FAIL: TestSplitHandlerCmdStr/Windows_style_path (0.00s)
        horenso_test.go:348: should be [go run C:\testdata\reporter.go arg1 arg2] but: [go run C:testdatareporter.go arg1 arg2]
FAIL
exit status 1
FAIL    github.com/hiroakis/horenso     0.253s
```
- https://github.com/hiroakis/horenso/pull/2/commits/9f10576bfa9dea1a35f6bebbfadb54cd3cab8220: Fix the issue.

Songmu commented:
Thanks for your pull request. This contribution is nice because windows support is not enough now.

However, simply using `strings.Fields` can not deal with the following cases.

```console
% horenso - r 'sh - c "path/to/cmd -opt1 -opt2"'
```

I am using the shellquote library for that, but it seems to have a bad influence in the Windows environment.

It is better to implement another splitting command string way for windows. Could you do it?

hiroakis commented:
Okay. I'll do that. I'm thinking of switching by runtime.GOOS.

```go
switch runtime.GOOS {
case "windows":
  strings.Split()
default:
  shellquote.Split()
}
```

Anyway, I'll update my PR.

hiroakis commented:
@Songmu Updated. Could you take a look?

Songmu commented:
Thanks. Released as v0.9.1.
